### PR TITLE
Traduction of few comments and changes in header of DataGenerator.py

### PR DESCRIPTION
**Pull request by Rayn2402**
Please do not remove the branch after accepting pull request, there is more editing coming in this branch soon.

### CODE REVIEW

File: Code/DataGenerator.py [hunk 1]
```diff
@@ -1,110 +1,129 @@
...
+        
+        # We convert our data points to cartesian coordinates
         data = np.vstack([self.polar_to_cart(radius, angle)]).T
+        
         return data, classes.astype(dtype='int32')
 
     def n_spike(self, nb_data, noise, distance, amplitude):
+        
         """
                Génère des données alatoires de type N-circle où N représente le nombre de classes. Les données sont
```
Comment (jodoin) [inline Code/DataGenerator.py]: traduire en anglais.

File: Code/DataGenerator.py [hunk 2]
```diff
@@ -141,6 +159,7 @@ def n_spike(self, nb_data, noise, distance, amplitude):
         return data, classes.astype(dtype='int32')
 
     def generer_donnees(self, modele='nCircle', noise=0.5, distance=5, amplitude=1):
```
Comment (jodoin) [inline Code/DataGenerator.py]: en anglais!

**Review by AleAyotte — APPROVED** [review]
The translation of comments is completed.

**Comment by AleAyotte** [comment]
I made a mistake Friday. Instead of creating another branch and create a pull request to merge with this current branch, i just made a new commit on the current branch. This will not happen twice.
So, if Rayn2402 can review and approve my commit, we will be able to merge.

**Pull request merged**